lib/heap/index.js

```diff
@@ -25,9 +25,17 @@ var Heap = module.exports = integration('Heap')
  */
 
 Heap.prototype.initialize = function(page){
-  window.heap=window.heap||[];window.heap.load=function(a){window._heapid=a;var d=function(a){return function(){window.heap.push([a].concat(Array.prototype.slice.call(arguments,0)));};},e=["identify","track"];for (var f=0;f<e.length;f++)window.heap[e[f]]=d(e[f]);};
-  window.heap.load(this.options.appId);
-  this.load(this.ready);
+  window.heap = window.heap || [];
+  window.heap.load = function(t, e) {
+    window.heap.appid = t, window.heap.config = e;
+    var o = function(t) {
+      return function() {
+        heap.push([t].concat(Array.prototype.slice.call(arguments, 0)))
+      }
+    };
+    p = ["identify", "track"];
+    for (c = 0; c < p.length; c++) heap[p[c]] = o(p[c])
+  };
 };
```

Comment:
Cool! Looks like the tests are failing on style things in the snippet - you can wrap it in `//jscs:disable` `//jscs:enable` if you don't want to alter it, otherwise you can have a look on circle for the issues.